<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIprogrammingClass</class>
 <widget class="QMainWindow" name="GUIprogrammingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIprogramming</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>파일 이름</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>형식</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>생성 시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>인물</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>장소</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이벤트</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>수정</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;w&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>q</string>
    </property>
    <property name="styleSheet">
     <string notr="true">e</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_Person">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QTableWidget" name="table_Person">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>211</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Child</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Location">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QTableWidget" name="table_Location">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>211</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Child</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Event">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
     <widget class="QTableWidget" name="table_Event">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>211</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Child</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>파일 이름으로 검색</string>
     </property>
     <property name="buddy">
      <cstring>line_Msearch</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Msearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="line_person">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_location">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_event">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_event">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>이벤트</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_location">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>장소</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_person">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>인물</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>상세검색</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableW_search">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>파일 이름</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="table_FP">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>좋아하는 사진</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="table_FV">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>좋아하는 동영상</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>좋아요</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>498</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUIprogramming.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>open_Add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellPressed(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>select_Row(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>display_content(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>open_Replace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>open_delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>search_con()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_Person</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>open_group(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>709</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_Event</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>open_group(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_Location</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>open_group(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>search_advanced()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_person</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_person</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_location</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_location</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_event</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_event</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableW_search</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>display_content_g(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>add_favorite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>delete_favorite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_FP</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>selet_FRow(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_FV</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>selet_FRow(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_FP</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>display_content_f(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>716</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_FV</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIprogrammingClass</receiver>
   <slot>display_content_f(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_Add()</slot>
  <slot>select_Row(int,int)</slot>
  <slot>display_content(int,int)</slot>
  <slot>open_Replace()</slot>
  <slot>open_delete()</slot>
  <slot>search_con()</slot>
  <slot>open_group(int,int)</slot>
  <slot>search_advanced()</slot>
  <slot>display_content_g(int,int)</slot>
  <slot>add_favorite()</slot>
  <slot>delete_favorite()</slot>
  <slot>selet_FRow(int,int)</slot>
  <slot>display_content_f(int,int)</slot>
 </slots>
</ui>
